### PR TITLE
Enable passing a ref to SearchMultiSelect input

### PR DESCRIPTION
**Pull request by edrpls**
This is a simple PR that allows passing a ref to be used in the SearchMultiSelect input, this is helpful to auto focus on the component after rendering.

### CODE REVIEW

File: _stories/molecules/SearchMultiSelect/README.md [hunk 1]
```diff
@@ -52,30 +62,30 @@ key        | uniquely identifies each options (key must be \`unique\`)
...
+| size        | size for the input, only md or sm are allowed {String}                                                                                                                                                                   |
+| searchKeys  | flag to signal if the search should also match against object keys {Boolean}                                                                                                                                             |
+| multiTerm   | flag to signal if the search should match against multiple terms {Boolean}                                                                                                                                               |
+| termDivider | string or regexp used to divide the search term, defaults to /[ ,]+/ (empty space and comma) {String OR RegExp}                                                                                                          |
+| inputRef    | ref applied to the input, useful for setting focus {function                                                                                                                                                             | object} |
```
Comment (w-dayrit) [inline _stories/molecules/SearchMultiSelect/README.md]: Table is broken here

Comment (edrpls) [reply _stories/molecules/SearchMultiSelect/README.md]: That's was automatically done by prettier. Looks fine when rendered.
screenshot_20180801_163555

Comment (w-dayrit) [reply _stories/molecules/SearchMultiSelect/README.md]: Oh I'm seeing this:
screen shot 2018-08-01 at 2 58 36 pm

Comment (edrpls) [reply _stories/molecules/SearchMultiSelect/README.md]: OH! my bad, yup, I'll fix it, thanks!

File: _stories/molecules/SearchMultiSelect/index.js [hunk 2]
```diff
@@ -12,6 +12,14 @@ class TestSearchMultiSelect extends React.Component {
         names: namesList
     };
 
+    componentDidMount() {
```
Comment (jscottsmith) [inline _stories/molecules/SearchMultiSelect/index.js]: Just a thought, we could also add a knob button to set focus when clicked.

Comment (edrpls) [reply _stories/molecules/SearchMultiSelect/index.js]: Done

File: _stories/molecules/SearchMultiSelect/README.md (hunk 1 above)
Comment (edrpls) [reply _stories/molecules/SearchMultiSelect/README.md]: Fixed

**Pull request merged**